fetools-app/src/components/TableGeneratorComponents/OptionsBoxTable.jsx

```diff
@@ -0,0 +1,226 @@
+import InputColor from './InputColor';
+import { handleOptions } from './TableGeneratorFN';
+
+import { TableInputs } from './TableInputs';
+import Icon from '../Icon';
+import { ToolPane } from '../ToolsLayout/Sections';
+
+export const OptionsBoxTable = ({ tableConfig, setTableConfig }) => {
+  return (
+    <ToolPane title="Options" icon="tune" isPrimary={true}>
+      <div className="flex flex-col gap-8 text-sm font-bold md:aspect-video lg:justify-evenly">
+        <section className="layout flex flex-wrap justify-between gap-y-4 [&>*]:w-[48%]">
+          <h3 className="border-b pb-1 border-[#D9D9D9] text-[18px] font-semibold min-w-full">
+            Layout
+          </h3>
+
+          <TableInputs
+            label="Rows"
+            icon="table_rows"
+            range={{ min: 2, max: 50 }}
+            ShowUnits={false}
+            tableConfig={tableConfig}
+            setTableConfig={setTableConfig}
+            prop="dimensions"
+            delay={true}
+          />
+
+          <TableInputs
+            label="Columns"
+            icon="view_week"
+            range={{ min: 1, max: 50 }}
+            ShowUnits={false}
+            tableConfig={tableConfig}
+            setTableConfig={setTableConfig}
+            prop="dimensions"
+            delay={true}
+          />
+          <TableInputs
+            label="Vertical Cell Padding"
+            icon="padding"
+            range={{ min: 0, max: 100 }}
+            tableConfig={tableConfig}
+            setTableConfig={setTableConfig}
+            prop="verticalCellPading"
+            delay={true}
+          />
+
+          <TableInputs
+            label="Horizontal Cell Padding"
+            icon="padding"
+            range={{ min: 0, max: 100 }}
+            tableConfig={tableConfig}
+            setTableConfig={setTableConfig}
+            prop="horizontalCellPading"
+          />
+          <TableInputs
+            label="Table Width"
+            icon="width"
+            range={{ min: 0, max: 1024 }}
```

Comment:
This probably needs to have a wider range. Also, all of these sliders need to be able to respond correctly when using rem/em units.